cluster/aws/config-default.sh

```diff
@@ -43,6 +43,7 @@ MINION_SCOPES=""
 POLL_SLEEP_INTERVAL=3
 PORTAL_NET="10.0.0.0/16"
 MASTER_IP_RANGE="${MASTER_IP_RANGE:-10.246.0.0/24}"
+CONTAINERS_CIDR=10.244.0.0/24
```

Comment:
You're absolutely right - nice spot.  Turns out I had made a second dumb mistake, which cancelled this one out: I got the name of the flag wrong when I passed it to kube-proxy, so it was going with the default, which is /16.  I will fix - thanks!